Add example and fix pep8 of find_fruit_veg (#82)
UBC-MDS/mealprep: README.Rmd

### `find_fruits_veg()`

> find_fruits_veg() will help you find the wanted columns index. 

Reviewer: ```suggestion
find_fruits_veg() will help you find the column indices for either numerical or categorical variables in your data frame. 
```

> The example below shows how to use find_fruits_veg() to find categorical columns' index of a toy data frame.

Reviewer: ```suggestion
The example below shows how to use find_fruits_veg() to find the categorical columns' index of a toy data frame.
```

First, load the required packages.

``` python
>>> from mealprep import mealprep
>>> import pandas as pd
```

> 

Reviewer: ```suggestion

Then, apply the find_fruits_veg() function to the data frame.
```
